x-pack/plugins/lens/public/xy_visualization/axis_settings_popover.tsx

```diff
@@ -236,6 +286,178 @@ export const AxisSettingsPopover: React.FunctionComponent<AxisSettingsPopoverPro
           />
         </>
       )}
+      {localExtent && setExtent && (
+        <>
+          <EuiSpacer size="s" />
+          <EuiFormRow
+            display="rowCompressed"
+            fullWidth
+            label={i18n.translate('xpack.lens.xyChart.axisExtent.label', {
+              defaultMessage: 'Bounds',
+            })}
+            helpText={
+              hasBarOrAreaOnAxis && localExtent.mode !== 'custom'
```

Comment:
Is it necessary to hide this text when the user has selected custom mode? I think it still has value regardless if the user is in custom mode or not. Possible to change?

```suggestion
              hasBarOrAreaOnAxis
```